code/modules/wiremod/shell/scanner_gate.dm

```diff
@@ -1,6 +1,6 @@
 /obj/structure/scanner_gate_shell
 	name = "circuit scanner gate"
-	desc = "A gate able to perform mid-depth scans on any organisms who pass under it."
+	desc = "A gate able to perform mid-depth scans on any organisms who pass under it. Requires to be secured to work."
```

Comment:
```suggestion
	desc = "A gate able to perform mid-depth scans on any organisms who pass under it. Needs to be secured to work."
```
Sounds more human this way

Reply:
Good thing you're here because i ain't no human